modules/serverless-kafka-broker-configmap.adoc

```diff
@@ -54,6 +60,9 @@ data:
   default.topic.partitions: "10"
   default.topic.replication.factor: "3"
   bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092"
+  triggers.consumergroup.template: "{% raw %}"knative-trigger-{{ .Namespace }}-{{ .Name }}-{{ .MyCustomValue}}"{% endraw %}"
+  brokers.topic.template: "{% raw %}"knative-broker-{{ .Namespace }}-{{ .Name }}-{{ .MyCustomValue}}"{% endraw %}"
+  channels.topic.template: "{% raw %}"messaging-kafka.{{ .Namespace }}.{{ .Name }}-{{ .MyCustomValue}}"{% endraw %}"
```

Comment:
This is not the right configmap for this feature, @Cali0707 isn't it in `config-kafka-features`?

Reply:
To be precise, this needs to be configured on the KnativeKafka resource by an "administrator"

Reply:
yep this is in `config-kafka-features` @pierDipi - good catch, thanks!

Reply:
@pierDipi Thanks, fixed